### ISSUE TITLE
Allow Close Project When Something Goes Wrong

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/RaveAddIn#136]
Currently, when something goes wrong loading a project a partial project tree node gets added. There's no "Close" option in the right context menu and so this half-complete project node gets left in the tree and the user is unable to get rid of it. Any subsequent attempts to reload the same project do nothing because RAVE detects the presence of the project already in the tree and does nothing.

Add "close project" right click menu option so that users can get rid of a broken project tree, (edit the business or project XML if needed) and try again.

**Comment by joewheaton** [comment]
Do you think we still need this @philipbaileynar? If so, do we also need to copy this and make issue in QRAVE?

**Comment by philipbaileynar** [comment]
Yes we need it in ArcRAVE. I don't know about QRAVE. Leave this ticket as it is for now. Don't create a QRAVE ticket until it becomes a problem in that software.
